KOGITO-3894: Anonymous navigation should be allowed when auth is not enabled (#3123)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/bpmn/chap-kogito-developing-process-services.adoc

IMPORTANT: This procedure applies only when you are using a locally cloned copy of the https://github.com/kiegroup/kogito-apps/tree/master/management-console[{PRODUCT} Management Console] repository in GitHub.

> NOTE: For testing purposes, Management Console also provides a Test User System that can be enabled by using the `kogito.test.user-system.enabled=true`.

Reviewer: There is an Important note available above this, therefore I would suggest if we can start the sentence with "Note that". wdyt?
```suggestion
Note that for testing purposes, the {PRODUCT} Management Console also provides a test user system, which is enabled using the `kogito.test.user-system.enabled=true`.
```

IMPORTANT: This procedure applies only when you are using a locally cloned copy of the https://github.com/kiegroup/kogito-apps/tree/master/task-console[{PRODUCT} Task Console] repository in GitHub.

> NOTE: For testing purposes, Task Console also provides a Test User System that can be enabled by using the `kogito.test.user-system.enabled=true`.

Reviewer: Similarly, here how about starting the sentence with "Note that"
```suggestion
Note that for testing purposes, the {PRODUCT} Task Console also provides a test user system, which is enabled using the `kogito.test.user-system.enabled=true`.
```
